packages/suite/src/support/suite/useConnectPopupDesktop.tsx

```diff
@@ -55,7 +68,7 @@ export const useConnectPopupDesktop = () => {
                 desktopApi.removeAllListeners('connect-popup/cancel');
             }
         };
-    }, [dispatch]);
+    }, [dispatch, experimentalFeatures]);
```

Comment:
init is now called everytime `experimentalFeatures` changes? doesn't it also mean  that you would register listners multiple times?

Reply:
True, adding a check https://github.com/trezor/trezor-suite/pull/18702/commits/e25e526d68a418da258b1e61799b70fdea7df9f3